stacks/openfaas/uninstall.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+set -e
+
+################################################################################
+# chart
+################################################################################
+STACK="openfaas"
+NAMESPACE="openfaas"
+
+
+helm uninstall "$STACK" \
+  --namespace "$NAMESPACE"
+
+helm uninstall ingress-nginx \
+  --namespace "$NAMESPACE"
+
+helm uninstall cert-manager \
+  --namespace "$NAMESPACE"
```

Comment:
This worries me a bit, sure they may be needed for opensaas to run, but should they be required by another install... would that not break that integration as well?

Reply:
ingress-nginx and cert-manager has been removed and advised the user to install them from the market-place